src/backend/app/Services/AgentTransactionService.php

```diff
@@ -54,15 +54,15 @@ static function ($q) use ($agentId) {
         return $transactions;
     }
 
-    public function getById(int $agentId, ?int $customerId = null): ?Transaction {
+    public function getByCustomerId(int $agentId, ?int $customerId = null): Collection|LengthAwarePaginator {
```

Comment:
We might want to pass the pagination params for this to work properly.

Reply:
Here I just restore the functionality to the way it was before. I think without parameter it just takes the default pagination.

Reply:
Make sense, I suppose implementing the pagination is a separate task.